### PR TITLE
fix: formatted message

### PR DESCRIPTION
**Pull request by vishdha**
420_Oasis_ACC_SINV_2020_00003 (1)

### CODE REVIEW

File: erpnext/selling/doctype/customer/customer.py [hunk 1]
```diff
@@ -330,8 +330,8 @@ def check_credit_limit(customer, company, ignore_outstanding_sales_order=False,
 				if frappe.db.exists("Employee", {"prefered_email": user})]
 
 			if credit_controller_users:
-				message = "Please contact any of the following users to extend the credit limits for {0}:\n{1}".format(
-					customer, "\n".join(credit_controller_users).replace("<", "(").replace(">", ")"))
+				message = "Please contact any of the following users to extend the credit limits for {0}:<br><br><li>{1}</li>".format(
```
Comment (Alchez) [inline erpnext/selling/doctype/customer/customer.py]: I think this should be 
```suggestion
				message = "Please contact any of the following users to extend the credit limits for {0}:<br><br><ul>{1}</ul>".format(
```

File: erpnext/selling/doctype/customer/customer.py [hunk 2]
```diff
@@ -330,8 +330,8 @@ def check_credit_limit(customer, company, ignore_outstanding_sales_order=False,
...
+					customer, '[li]'.join(credit_controller_users).replace("<", "(").replace(">",")").replace("[","<").replace("]",">"))
```
Comment (Alchez) [inline erpnext/selling/doctype/customer/customer.py:336]: Why are we replacing "[]" to "<>"?

Comment (vishdha) [reply erpnext/selling/doctype/customer/customer.py:336]: .replace("<", "(") this modifed html tag. thats why

Comment (Alchez) [reply erpnext/selling/doctype/customer/customer.py:336]: Are there are "[]" characters anyway? I don't see any

File: erpnext/selling/doctype/customer/customer.py [hunk 3]
```diff
@@ -326,16 +327,39 @@ def check_credit_limit(customer, company, ignore_outstanding_sales_order=False,
 		credit_controller_role = frappe.db.get_single_value('Accounts Settings', 'credit_controller')
 		if not credit_controller_role or credit_controller_role not in frappe.get_roles():
 			credit_controller_users = get_users_with_role(credit_controller_role or "Sales Master Manager")
+			credit_controller_users_list = [user for user in credit_controller_users
+				if frappe.db.exists("Employee", {"prefered_email": user})]
 			credit_controller_users = [get_formatted_email(user) for user in credit_controller_users
 				if frappe.db.exists("Employee", {"prefered_email": user})]
```
Comment (Alchez) [inline erpnext/selling/doctype/customer/customer.py]: If you're doing the above addition, change this to:

```suggestion
			credit_controller_users = [get_formatted_email(user) for user in credit_controller_users_list]
```

File: erpnext/selling/doctype/customer/customer.py [hunk 4]
```diff
@@ -326,16 +327,39 @@ def check_credit_limit(customer, company, ignore_outstanding_sales_order=False,
...
+						'args': {
+							'customer': customer,
+							'customer_outstanding': customer_outstanding,
+							'credit_limit': credit_limit,
+							'credit_controller_users_list': credit_controller_users_list
+					}
+				})
 			else:
 				message = "Please contact your administrator to extend the credit limits for {0}.".format(customer)
+				throw(_(message))
```
Comment (Alchez) [inline erpnext/selling/doctype/customer/customer.py]: Change this to a `frappe.throw` directly

File: erpnext/selling/doctype/customer/customer.py [hunk 5]
```diff
@@ -326,16 +327,39 @@ def check_credit_limit(customer, company, ignore_outstanding_sales_order=False,
...
 				if frappe.db.exists("Employee", {"prefered_email": user})]
 
 			if credit_controller_users:
-				message = "Please contact any of the following users to extend the credit limits for {0}:\n{1}".format(
-					customer, "\n".join(credit_controller_users).replace("<", "(").replace(">", ")"))
+				message = "Please contact any of the following users to extend the credit limits for {0}:<br><br><ul><li>{1}</li></ul>".format(
+					customer, '[li]'.join(credit_controller_users).replace("<", "(").replace(">",")").replace("[","<").replace("]",">"))
+
+				# if users has credit controller role set then send emails.
+				frappe.msgprint(message, title = "Message",
```
Comment (Alchez) [inline erpnext/selling/doctype/customer/customer.py]: Change "Message" to "Notify"

**Review by Alchez — CHANGES_REQUESTED** [review]

**Review by Alchez — APPROVED** [review]

**Pull request merged**